beta/nncf/tensorflow/quantization/quantizers.py

```diff
@@ -292,13 +305,10 @@ def get_quantizer_config(self) -> QuantizerConfig:
             per_channel=self.per_channel
         )
 
-    def get_config(self):
-        qspec = TFQuantizerSpec(num_bits=self.num_bits,
-                                mode=QuantizationMode.SYMMETRIC,
-                                signedness_to_force=self.signedness_to_force,
-                                narrow_range=self.narrow_range,
-                                per_channel=self.per_channel)
-        return deepcopy(qspec.__dict__)
+    def get_config(self, keep_op_name=True):
+        return super()._get_config(mode=QuantizationMode.SYMMETRIC,
+                                  signedness_to_force=self.signedness_to_force,
+                                  keep_op_name=keep_op_name)
```

Comment:
Can you please refactor this according to the style guide?

Reply:
Done